hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/util/TestJarFinder.java

```diff
@@ -39,14 +38,6 @@
 
 public class TestJarFinder {
 
-  @Test
```

Comment:
test should be replaced with anothe class we know is there, maybe one of the junit ones

Reply:
Done